cdn/signUrl/SignUrlSample/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright 2018 Google Inc. All rights reserved.
```

Comment:
Copyright ditto.

Reply:
fixed. Apparently, we have different copyright formats for different languages. Previous was copied from another sample (not dotnet).